### PR TITLE
538 sections introduction

### PR DESCRIPTION
**Pull request by adammahmood**
### What is the context of this PR?
 - Adds section introduction

### How to review 
- Run tests
- Ensure the schema makes sense

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -44,12 +44,15 @@ type Section {
...
   questionnaire: Questionnaire
   position: Int!
-  introduction: Introduction
+  introductionTitle: String
+  introductionContent: String
+  introductionEnabled: Boolean!
+  introduction: Introduction @deprecated(reason: "Use Section fields instead")
 }
 
 type Introduction {
```
Comment (SamGodwin2) [inline index.js]: Think we're also going to need to deprecated this type so we can clean it up later.

File: index.js [hunk 2]
```diff
@@ -498,23 +504,23 @@ input DuplicateSectionInput {
 }
 
 input CreateIntroductionInput {
```
Comment (SamGodwin2) [inline index.js]: Not sure but I have a feeling these Input types will need to be deprecated as well so they can be deleted later?

File: index.js (hunk 1 above)
Comment (rmccar) [reply index.js]: Removed on deprecated fields branch

File: index.js (hunk 2 above)
Comment (rmccar) [reply index.js]: Removed on deprecated fields branch

**Pull request merged**